spring-gemfire/src/main/java/org/springframework/geode/config/annotation/ClusterAvailableConfiguration.java

```diff
@@ -161,7 +160,7 @@ protected String getRuntimeEnvironmentName() {
 
 		@Override
 		protected boolean isCloudPlatformActive(@NonNull Environment environment) {
-			return environment != null && CloudPlatform.KUBERNETES.isActive(environment);
+			return CloudPlatform.KUBERNETES.isActive(environment);
```

Comment:
Are we sure that the `environment != null` is cleanly handled in the `isActive` method?